content/about/about_people.rst

```diff
@@ -590,6 +590,30 @@ https://github.com/ElijahCaine
 
   <br/>
 
+`Levi Willmeth`_
+----------------
+
+.. image:: /images/lwillmeth.jpg
+    :scale: 50%
+    :align: right
+    :alt: Levi Willmeth
+
+.. class:: no-breaks
+
+  **Student Intern | Something something**
+
+Levi will be working at OSL over the summer as a student intern.  He is excited to be working with a group on open source projects.  He has two very energetic daughters and enjoys convincing them the world works in ridiculous ways.  He also enjoys flying remote controlled quadcopters, building small electronics projects, and contributing to the OSU/LBCC Space Club.
```

Comment:
"Levi is working" (Since you've started it's no longer future tense). "the OSL" (Cause this is how we do). *Second sentence* If you have a specific project or skill set you can name, that would be nice. Otherwise I don't think this is necessary.

Also, if you hit enter when your text hits that line on the far right side of the computer screen, that's where the 80 character mark is and the program will then accept your text.